Gekko/DataProviders/BTCTradeUA/TradingPlatformAccessibilityController.swift

```diff
@@ -0,0 +1,41 @@
+//  Created by Aleksandr Saliienko on 4/12/18.
+//  Copyright © 2018 Sergii Mykhailov. All rights reserved.
+//
+
+import Foundation
+
+protocol ChangeServerAccessibilityDelegate {
+    func setupServerErrorView()
+    func removeServerErrorView()
+}
+
+class TradingPlatformAccessibilityController {
+    
+    public var delegate:ChangeServerAccessibilityDelegate?
```

Comment:
Delegate should be stored by weak reference. Otherwise there will be retain cycle and we will get memory leak:
public weak var delegate